python/cudf/cudf/tests/test_rolling.py

```diff
@@ -97,17 +89,24 @@ def test_rolling_dataframe_basic(data, agg, nulls, center):
             got = getattr(
                 gdf.rolling(window_size, min_periods, center), agg
             )().fillna(-1)
-            try:
-                assert_eq(expect, got, check_dtype=False)
-            except AssertionError as e:
-                if agg == "count" and len(pdf) > 0:
-                    pytest.xfail(reason="Differ from pandas behavior here")
-                else:
-                    raise e
+            assert_eq(expect, got, check_dtype=False)
 
 
 @pytest.mark.parametrize(
-    "agg", ["sum", pytest.param("min"), pytest.param("max"), "mean", "count"]
+    "agg",
+    [
+        "sum",
+        pytest.param("min"),
+        pytest.param("max"),
+        "mean",
+        pytest.param(
+            "count",  # Does not follow similar conventions as
```

Comment:
Since we are removing `count` from this test because it does't match Pandas behaviour, should we add another test for `count` that checks against what _we_ consider to be correct behaviour?

Reply:
I've added an explicit test for offset rolling windows, for count.